Misc/NEWS.d/next/Library/2024-12-21-03-20-12.gh-issue-128131.QpPmNt.rst

```diff
@@ -0,0 +1 @@
+Support forward seek in uncompressed unencrypted ZipExtFile
```

Comment:
```suggestion
Support forward seek in uncompressed unencrypted :class:`!zipfile.ZipExtFile`.
```

We don't have a documented entry for `ZipExtFile` so we can only suppress the link, but it's good to indicate that it's part of the zipfile module.